api/s2n.h

```diff
@@ -1210,9 +1213,9 @@ extern int s2n_config_set_ticket_decrypt_key_lifetime(struct s2n_config *config,
  */
 S2N_API
 extern int s2n_config_add_ticket_crypto_key(struct s2n_config *config,
-                                            const uint8_t *name, uint32_t name_len,
-                                            uint8_t *key, uint32_t key_len,
-                                            uint64_t intro_time_in_seconds_from_epoch);
+        const uint8_t *name, uint32_t name_len,
+        uint8_t *key, uint32_t key_len,
+        uint64_t intro_time_in_seconds_from_epoch);
 /**
```

Comment:
Also a pretty strange choice. Did clang-format do this?

Reply:
Fairly certain. What is going on here is -> column limit 0 means that we accept the developer's choice with regard to newlines. So we don't re flow the arguments. But we decided for other reasons to indent by a fixed amount. So we move those args back. (This is consistent with what happened elsewhere.) I'm pretty sure if I manually wrap it to 120 and re-format it will look better

Reply:
Here I made some different choices choices about wrapping -- let me know if you'd prefer it way it was -- easy to fix.